.github/workflows/comment.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     if: github.event.workflow_run.event == 'pull_request'
     steps:
       - uses: actions/checkout@v3
-      - uses: canonical/operator-workflows/.github/actions/comment_pr
+      - uses: canonical/operator-workflows/.github/actions/comment_pr@main
```

Comment:
I think you can use relative path here, like `./.github/actions/comment_pr`
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-an-action-in-the-same-repository-as-the-workflow

Reply:
I'm checking out the indico repository and the file doesn't exist in there. In the next PR I'll move this action to a new repository,  so I'd leave it as it is for now if that works for you